index-pl.html

```diff
@@ -535,30 +535,30 @@ <h2>How To Stop An Epidemic</h2>
   <div class="container">
 
     <p>
-      To stop the spread of COVID-19, we need to keep R0 beneath 1.
-      When this happens, on average, a contagious person will infect less than one person, and the epidemic will grind to a halt.
+      Aby zatrzymać rozprzestrzenianie się COVID-19, musimy utrzymać wartość R0 poniżej 1.
+      Jeśli uda się to osiągnąć, średnio jedna zakażona osoba zarazi mniej niż jedną osobę a epidemia powoli się zatrzyma.
     </p>
 
     <p>
-      So how many people need to wear a 50% effective mask to stop the spread of COVID-19?
-      What if masks were 75% effective?
-      Or 90% effective?
-      This interactive lets you predict answers to these questions.
+      W takim razie, ile osób musi nosić 50-procentowo skuteczne maski abyśmy zatrzymali rozprzestrzenianie się COVID-19?
+      Co stałoby się gdyby maski były skuteczne w 75%?
+      Lub skuteczne w 90%?
+      Ten interaktywny panel pozwala Wam zobaczyć odpowiedzi na te pytania.
     </p>
 
     <aside>
 
-      <span @click="expandaside2 = true;">An important caveat: people don't usually mix randomly. 
-        <span v-if="!expandaside2"> (Click to expand)</span>
+      <span @click="expandaside2 = true;">Istotne zasrzeżenie: ludzie zwykle nie spotykają się zupełnie przypadkowo. 
```

Comment:
1. Literówka: "zasrzeżenie" 
2. przypadkowo -> losowo? Niby to samo, ale w tym wypadku "przypadkowo" brzmi jakby chodziło i randki : )